### PR TITLE
Migrate sample project to AndroidX

### PR DESCRIPTION
**Pull request by 4brunu**

### CODE REVIEW

File: Sample/Sample.csproj [hunk 1]
```diff
@@ -106,10 +106,10 @@
     <Folder Include="Resources\drawable\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Android.Support.Design" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.CustomTabs" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.3.1" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.7.0" />
+    <PackageReference Include="Xamarin.AndroidX.AppCompat">
```
Comment (gsgou) [inline Sample/Sample.csproj:110]: if you like you can use following syntax
<PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.3.1.1" />

File: Sample/MainActivity.cs [hunk 2]
```diff
@@ -37,6 +32,7 @@
 using PDFTextStripper = Com.Tom_roush.Pdfbox.Text.PDFTextStripper;
 using PDFBoxResourceLoader = Com.Tom_roush.Pdfbox.Util.PDFBoxResourceLoader;
 using ImageType = Com.Tom_roush.Pdfbox.Rendering.ImageType;
+using AndroidX.AppCompat.App;
```
Comment (gsgou) [inline Sample/MainActivity.cs:35]: if you like you can move this below
using Android.Widget;

**Review by gsgou — APPROVED** [review]

File: Sample/Sample.csproj (hunk 1 above)
Comment (gsgou) [reply Sample/Sample.csproj:110]: When VS updates versions keeps the current syntax but by adding new package breaks it.
Its an OCD to prefer the shorter :D Thanks.

File: Sample/MainActivity.cs (hunk 2 above)
Comment (4brunu) [reply Sample/MainActivity.cs]: Done 👍

File: Sample/Sample.csproj (hunk 1 above)
Comment (4brunu) [reply Sample/Sample.csproj]: Done 👍

**Review by gsgou — APPROVED** [review]

**Pull request merged**